pkg/netutil/isolate_linux.go

```diff
@@ -40,3 +40,24 @@ func RecoverPort(port int) error {
 	_, err := exec.Command("/bin/sh", "-c", cmdStr).Output()
 	return err
 }
+
+// AddLatency adds latency in millisecond scale with +/-10ms random variations.
+func AddLatency(ms int) error {
+	cmdStr := fmt.Sprintf("sudo tc qdisc add dev eth0 root netem delay %dms 10ms distribution normal", ms)
+	_, err := exec.Command("/bin/sh", "-c", cmdStr).Output()
+	if err != nil {
+		// the rule has already been added. Overwrite it.
+		cmdStr = fmt.Sprintf("sudo tc qdisc change dev eth0 root netem delay %dms 10ms distribution normal", ms)
+		_, err = exec.Command("/bin/sh", "-c", cmdStr).Output()
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// RemoveLatency resets netem configurations.
```

Comment:
the comment is bad. it seems like we want to hide the details of netem. so we should not mention it.